oozie/src/test/java/org/apache/falcon/oozie/process/OozieProcessWorkflowBuilderTest.java

```diff
@@ -192,17 +191,47 @@ public void testDefCoordMap(Process process, COORDINATORAPP coord) throws Except
         HashMap<String, String> wfProps = getWorkflowProperties(fs, coord);
         assertEquals(wfProps.get("mapred.job.priority"), "LOW");
         List<Input> inputs = process.getInputs().getInputs();
-        assertEquals(props.get(WorkflowExecutionArgs.INPUT_NAMES.getName()), inputs.get(0).getName() + "#" + inputs
-            .get(1).getName());
+        assertEquals(props.get(WorkflowExecutionArgs.INPUT_NAMES.getName()),
+                inputs.get(0).getName() + "#" + inputs.get(1).getName());
 
-        verifyEntityProperties(process, cluster,
-                WorkflowExecutionContext.EntityOperations.GENERATE, wfProps);
+        verifyEntityProperties(process, cluster, WorkflowExecutionContext.EntityOperations.GENERATE, wfProps);
         verifyBrokerProperties(cluster, wfProps);
 
         assertLibExtensions(fs, coord, EntityType.PROCESS, null);
     }
 
     @Test
+    public void testCoordinatorThrottleMin() throws Exception {
+        Process process = ConfigurationStore.get().get(EntityType.PROCESS, "clicksummary");
+
+        process.setFrequency(Frequency.fromString("hours(24)"));
+        process.setTimeout(Frequency.fromString("hours(20)"));
+
+        OozieEntityBuilder builder = OozieEntityBuilder.get(process);
+        Path bundlePath = new Path("/falcon/staging/workflows", process.getName());
+        builder.build(cluster, bundlePath);
+        assertTrue(fs.exists(bundlePath));
+
+        BUNDLEAPP bundle = getBundle(fs, bundlePath);
+        assertEquals(EntityUtil.getWorkflowName(process).toString(), bundle.getName());
+        assertEquals(1, bundle.getCoordinator().size());
+        assertEquals(EntityUtil.getWorkflowName(Tag.DEFAULT, process).toString(),
+                bundle.getCoordinator().get(0).getName());
+        String coordPath = bundle.getCoordinator().get(0).getAppPath().replace("${nameNode}", "");
+        List<CONFIGURATION.Property> props = bundle.getCoordinator().get(0).getConfiguration().getProperty();
+        for (CONFIGURATION.Property prop : props) {
+            if (prop.getName().equals("oozie.libpath")) {
+                Assert.assertEquals(prop.getValue().replace("${nameNode}", ""), process.getWorkflow().getLib());
+            }
+        }
+
+        COORDINATORAPP coord = getCoordinator(fs, new Path(coordPath));
+        testDefCoordMap(process, coord);
+
+        assertEquals(coord.getControls().getThrottle(), "12");
```

Comment:
I suppose the question is what do we think is important for the test to show? Because the result of this behaviour is to select the largest of **two values** I will include a test for each.